docs/writing-tests/snapshot-testing.md

```diff
@@ -8,9 +8,56 @@ Storybook is a helpful tool for snapshot testing because every story is essentia
 
 ![Example Snapshot test](./snapshot-test.png)
 
-## Setup Storyshots
+## Set up test runner
 
-[Storyshots](https://storybook.js.org/addons/@storybook/addon-storyshots/) is the official Storybook addon that enables snapshot testing, powered by [Jest](https://jestjs.io/docs/getting-started).
+Storybook test runner turns all of your stories into executable tests. It is powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/).
+
+Follow the [setup instructions](./test-runner#setup) to install and run the test runner.
+
+Then, you can use the [test hook API](./test-runner#test-hook-api-experimental) to snapshot the HTML of each story:
+
+```js
+// .storybook/test-runner.js
+
+module.exports = {
+  async postRender(page, context) {
+    // the #storybook-root element wraps each story
+    const elementHandler = await page.$('#storybook-root');
+    const innerHTML = await elementHandler.innerHTML();
+    expect(innerHTML).toMatchSnapshot();
+  },
+};
+```
+
+When running in [stories.json mode](./test-runner#storiesjson-mode), tests are generated in a temporary folder and snapshots get stored alongside them. You will need to [`--eject`](./test-runner#configure) and configure a custom [`snapshotResolver`](https://jestjs.io/docs/configuration#snapshotresolver-string) to store them elsewhere, e.g. in your working directory:
```

Comment:
@yannbf — Should this be "index.json. mode"? The README uses that (and `--stories-json` isn't on the page), but the [test runner docs page](https://storybook.js.org/docs/react/writing-tests/test-runner#storiesjson-mode) apparently hasn't been updated.

If your answer is "yes" to the above, I'll update the test runner docs, too (unless you tell me not to).

Reply:
@yannbf ?

Reply:
Hey! Index-json is preferred, as is the new way forward. stories-json is still available and works for Storybook 6 (which does not have an index.json file), but will likely be removed in the future